kubernetes/kserve/kserve_wrapper/README.md

```diff
@@ -136,7 +136,7 @@ For v1 protocol
 The curl request for inference is as below:
 
 ```bash
-curl -H "Content-Type: application/json" --data @serve/kubernetes/kserve/kf_request_json/v1/mnist.json http://127.0.0.1:8080/v1/models/mnist:predict
+curl -H "Content-Type: application/json" --data @serve/kubernetes/kserve/kf_request_json/v1/mnist.json http://0.0.0.0:8080/v1/models/mnist:predict
```

Comment:
Please update to match the steps for getting the ingress host ip and port for running prediction, from here: 

https://kserve.github.io/website/0.11/modelserving/v1beta1/torchserve/#model-inference_1